vscode/package.json

```diff
@@ -1,26 +1,47 @@
 {
     "name": "c3-vscode",
-    "displayName": "c3-vscode",
-    "description": "C3 Language Support for VSCode",
-    "version": "0.0.2",
+    "displayName": "C3 Language Support for VSCode",
+    "description": "C3 Language Support for VSCode - Syntax Highlighting and Code Snippets",
+    "publisher": "C3Language",
+    "repository": "https://github.com/c3lang/editor-plugins.git",
+    "version": "0.0.3",
     "engines": {
-        "vscode": "^1.63.0"
+        "vscode": "^1.75.0"
     },
     "categories": [
-        "Programming Languages"
+        "Programming Languages",
+        "Snippets"
     ],
+    "icon": "images/icon.png",
+    "license": "MIT",
     "contributes": {
-        "languages": [{
-            "id": "c3",
-            "aliases": ["C3", "c3"],
-            "extensions": ["c3", "c3i"],
-            "configuration": "./language-configuration.json"
-        }],
-        "grammars": [{
-            "language": "c3",
-            "scopeName": "source.c3",
-            "path": "./syntaxes/c3.tmLanguage.json"
-        }],
+        "languages": [
+            {
+                "id": "c3",
+                "aliases": [
+                    "C3",
+                    "c3"
+                ],
+                "extensions": [
+                    "c3",
+                    "c3i"
+                ],
```

Comment:
Another extension is "c3t" which is used for compiler test cases.

Reply:
Fixed.